gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -425,6 +434,39 @@ public static List<Service> parseService(
         Transport.GRPC);
   }
 
+  private static boolean shouldIncludeMethodInGeneration(
```

Comment:
Can we make this a default scope method so we can test it directly? This would enabled us testing more scenarios like the non matching packages, it would also simplify the golden tests.